my-app/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You need to look at files, which you commit before the commit. This file is not connected to this project